### PR TITLE
Capture more commit diff comments and editor

### PR DESCRIPTION
**Pull request by Martii**
* Think this might work a little better in some other areas. Not entirely sure why it was stuck at `960px` before. Let me know if there are any issues/changes and thanks for the consideration. :)

NOTE:
* Needs `make.js` run since this is a build system :)

### CODE REVIEW

File: wide-github.css [hunk 1]
```diff
@@ -34,15 +34,20 @@ header .container-lg {
 .repository-content .timeline-new-comment { /* New Issue / issue comment form */
   max-width: 100% !important;
 }
-.repository-content .inline-comments .comment-holder { /* Diff / code comments */
-  max-width: 960px !important;
-}
-.repository-content .inline-comments .inline-comment-form-container {
-  max-width: 960px !important;
-}
```
Comment (Martii) [inline wide-github.css]: Sample for #34

**Comment by Martii** [comment]
Hang on a second... pasted .js into the .css

**Comment by Martii** [comment]
Thinks that's right now. Too tired atm. :)

File: wide-github.css [hunk 2]
```diff
@@ -34,15 +34,20 @@ header .container-lg {
...
+  "max-width: inherit !important;" +
+"}" +
+".repository-content .inline-comments .inline-comment-form {" +
+  "max-width: inherit !important;" +
+"}" +
+
+".repository-content #all_commit_comments .commit-comments-heading," +
+".repository-content #all_commit_comments .comment-holder {" +
+  "max-width: inherit !important;" +
+"}" +
```
Comment (Martii) [inline wide-github.css]: Resample for #34 (click View changes button if looking in the PR)

File: wide-github.css [hunk 3]
```diff
@@ -34,15 +34,20 @@ header .container-lg {
...
+".repository-content .inline-comments .inline-comment-form-container {" +
+  "max-width: inherit !important;" +
+"}" +
+".repository-content .inline-comments .inline-comment-form {" +
+  "max-width: inherit !important;" +
+"}" +
+
+".repository-content #all_commit_comments .commit-comments-heading," +
+".repository-content #all_commit_comments .comment-holder {" +
+  "max-width: inherit !important;" +
```
Comment (Martii) [inline wide-github.css]: Resample for #34 *(click View changes button if looking in the PR)* *(sorry for dupes GH is picky about showing in PR)*

File: wide-github.css [hunk 4]
```diff
@@ -35,13 +35,18 @@ header .container-lg {
...
+  max-width: inherit !important;
 }
 .repository-content .inline-comments .inline-comment-form {
-  max-width: 960px !important;
+  max-width: inherit !important;
+}
+
+.repository-content #all_commit_comments .commit-comments-heading,
+.repository-content #all_commit_comments .comment-holder {
+  max-width: inherit !important;
```
Comment (xthexder) [inline wide-github.css]: Which view is this for? I wasn't able to find anything out of place with just the inline comment changes.

Comment (Martii) [reply wide-github.css]: Click the View changes button in the PR and then go to the bottom GH editor. I'll attempt to back it out of my local copy and give a screenshot attached to this comment...

Comment (Martii) [reply wide-github.css]: Scratch that... it's at https://github.com/xthexder/wide-github/commit/46a4c85299099626ad16eca7d046173d1954a325 ... go to the bottom editor there.

Comment (xthexder) [reply wide-github.css]: Ah I see it now. Changes all look good, I'll run the make.js and merge this when I'm home from work.

**Comment by Martii** [comment]
Btw are you ever going to do [the webhook instructions](https://openuserjs.org/user/add/scripts#sync-script) for [this repository](https://github.com/Martii/UserScripts/settings/hooks)? *(I did a bunch of changes on OUJS just for this to assist you, and others too)*. Typically I prefer not to sync scripts to OUJS as you... but you are quite popular over there. :)

And I wouldn't get too bound to thinking about using `@downloadURL` because of the comment at https://github.com/greasemonkey/greasemonkey/issues/2836#issuecomment-364289675 . I'm not thrilled that it's being eol'd but I can understand the flip side of the coin. The webhook will keep your presentational presence up to date. I really appreciate your work on this script. I had one back in the USO days but it was way too cumbersome to maintain through all of their changes back then... then I discovered yours. :)

**Comment by xthexder** [comment]
I'll probably look at setting up webhooks and an auto-build system for this at some point, It will be more convienient to make changes if there's no extra commands to run.

**Pull request merged**